src/State/Hub.php

```diff
@@ -274,4 +291,20 @@ public static function setCurrent(self $hub): self
 
         return $hub;
     }
+
+    /**
+     * Gets the integration whose FQCN matches the given one if it's available on the current client.
+     *
+     * @param string $className The FQCN of the integration
+     *
+     * @return null|IntegrationInterface
+     */
+    public function getIntegration(string $className): ?IntegrationInterface
+    {
+        if ($client = $this->getClient()) {
```

Comment:
Please split this condition so that it explicitly check for `null` like it's done in the other methods